tutorials/vue/v-if-and-else.md

```diff
@@ -0,0 +1,50 @@
+To conditionally render something in vue, you must use `v-if` and `v-else` directive.
+Simply pass an expression to the `v-if` directive, and the block will render if the expression is true.
+`v-else` will render in the event none of the `v-if` directives render.
```

Comment:
"You can also use `v-else`, which will render if the `v-if` expression evaluates to a falsy value."